torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2900,7 +2900,10 @@ def _maybe_evaluate_static(self, expr: "sympy.Expr", *, unbacked_only: bool = Fa
             # Don't do anything if we don't have a nontrivial lower bound
             # Also don't do anything if we asked only to simplify unbacked
             # SymInt
-            if vr.lower == -sympy.oo or (unbacked_only and k in self.var_to_val):
+            if (
+                vr.lower < (-sys.maxsize - 1) // 2 or
```

Comment:
Why halfway through the negatives is your bound here?

Reply:
I mostly want to distinguish between "-sys.maxsize - 1" and "small negative numbers that aren't too negative". But I could probably also do vr.lower < 0 without too much trouble; happy to change it.

Reply:
The halfway boundary seems a bit surprising, but if it is what you're after that sounds ok. I don't have a strong opinion.